docs/howto/operate/manual-nfs-setup.md

```diff
@@ -0,0 +1,89 @@
+# Manually setup NFS Server for a cluster
+
+This guide describes the manual steps required for setting up a NFS server to store users' home directories on the hub.
+More information about the NFS Server can be found in {ref}`/topic/storage-layer`.
+
+## Deploy the host Virtual Machine
+
+We need to first deploy a small virtual machine with a persistent disk that will host the NFS server.
+Using `gcloud`, the command is:
+
+```bash
+gcloud compute instances create nfs-server-01 \
+  --image=ubuntu-2004-focal-v20210720 \
+  --image-project=ubuntu-os-cloud \
+  --machine-type=g1-small \
+  --boot-disk-device-name=nfs-server-01 \
+  --boot-disk-size=100GB \
```

Comment:
Let's note that the data is also in the boot disk here, and suggest people make it bigger if they need to?

Reply:
@sgibson91 feel free to self-merge after adding this